### PR TITLE
Allow using '-' as filename in -f to read csv from STDIN.  Fixes #95

### LINKED ISSUE #95: read from stdin
reckon is really nice, I had my banks' CSV export converted to ledger format in no time.

it would make my workflow a bit simpler if reckon supported passing in `-f -` to read the CSV from stdin.

### PR DESCRIPTION
**Pull request by benprew**

### CODE REVIEW

File: lib/reckon/app.rb [hunk 1]
```diff
@@ -394,6 +394,15 @@ def self.parse_opts(args = ARGV)
         opts.parse!(args)
       end
 
+      if options[:file] == '-'
+        unless options[:unattended]
+          raise "--unattended is required to use STDIN as CSV source."
+        end
+
+        puts "Reading csv from STDIN"
```
Comment (raboof) [inline lib/reckon/app.rb:402]: This was probably a debugging leftover?

Comment (benprew) [reply lib/reckon/app.rb:402]: Good catch, I removed it, thanks!

Comment (raboof) [reply lib/reckon/app.rb:402]: with https://github.com/cantino/reckon/commit/d88814f5f7f1fa479d310ceac8c4e3b7cee47313 this *looks* good to me. When I build this package pointing to that commit (I'm on nixpkgs) and run `echo "" | reckon -f - --unattended` I still get `Reading csv from STDIN`, but I'm probably just misunderstanding something on the nixpkgs side. I'll test again when 0.6.3 is out.

Comment (raboof) [reply lib/reckon/app.rb:402]: can confirm this is fixed on 0.7.1!

**Comment by benprew** [comment]
Great, thanks!

**Pull request merged**